curriculum/challenges/chinese/00-certificates/machine-learning-with-python-certificate/machine-learning-with-python-certificate.yml

```diff
@@ -1,6 +1,6 @@
 ---
 id: 5e46fc95ac417301a38fb935
-title: Python 与机器学习证书
+title: Machine Learning with Python Certificate
```

Comment:
This does not look, right? Aren't we going with "Certifications"? /cc @freeCodeCamp/i18n

Reply:
Looks the same as the [other one](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/00-certificates/back-end-development-and-apis-certificate/back-end-development-and-apis-certificate.yml#L3)s. Except it's been untranslated because of the rename. Wonder if we should have waited on merging these until the translations caught up with where they were before the rename.

Reply:
Seems, these aren't used. But we should standardize the convention to avoid tripping over.